### PR TITLE
📖 Updating QuickStart: command to create kind clusters

### PR DESCRIPTION
**Pull request by dumb0002**
This PR updates the creation of the Florin and Guilder cluster for inline(text)

## Related issue(s)

Fixes #

### CODE REVIEW

**Review by clubanderson — COMMENTED** [review]
/lgtm

File: docs/content/Getting-Started/quickstart.md [hunk 1]
```diff
@@ -87,15 +87,31 @@ workload and summarization aspirations removed.
...
-kind create cluster --name guilder --config  kubestellar/examples/guilder-config.yaml
+kind create cluster --name guilder --config - <<EOF
+kind: Cluster
+apiVersion: kind.x-k8s.io/v1alpha4
+nodes:
+- role: control-plane
+  extraPortMappings:
+  - containerPort: 8081
+    hostPort: 8096
+EOF
```
Comment (MikeSpreitzer) [inline docs/content/Getting-Started/quickstart.md:114]: The file in `examples` has two port mappings here, which supports users who go on to the extended example.  If the second port mapping is excluded here then users can not go on to the extended example.

Comment (dumb0002) [reply docs/content/Getting-Started/quickstart.md:116]: Fixed. I added the second port mapping for the guilder cluster

**Review by clubanderson — COMMENTED** [review]
/lgtm

**Review by MikeSpreitzer — APPROVED** [review]
/lgtm
Thanks

**Pull request merged**